infra/resources/prod/main.tf

```diff
@@ -117,6 +117,7 @@ module "function_apps" {
   application_insights_connection_string = data.azurerm_application_insights.common.connection_string
 
   revocation_queue_name                            = module.storage_accounts.revocation_queue_name_02.name
+  send_email_queue_name                            = module.storage_accounts.revocation_queue_name_02.name
```

Comment:
this is the wrong queue. you should create a new queue `send-email` if it doesn't already exist